fontforge/sfd.c

```diff
@@ -7895,6 +7891,10 @@ bool SFD_GetFontMetaData( FILE *sfd,
 	    d->last_base->scripts = bs;
 	d->last_base_script = bs;
     }
+    else if ( strmatch(tok,"StyleMap:")==0 )
+    {
+    gethex(sfd,(uint32 *)&sf->sfntRevision);
```

Comment:
I think I noticed it simultanouesly to you while looking at your previous comment, haha. Updated.